pkg/controllers/state/suite_test.go

```diff
@@ -1263,6 +1263,15 @@ var _ = Describe("Cluster State Sync", func() {
 		}
 		Expect(cluster.Synced(ctx)).To(BeFalse())
 	})
+	It("shouldn't consider the cluster state synced if a nodeclaim isn't launched", func() {
```

Comment:
I think this might be covered by a previous test case that syncs a bunch of nodes and then has a single node that doesn't have a provider id